src/config/index.js

```diff
@@ -26,10 +26,16 @@ const configuration = {
   CUSTOMER_SUPPORT_EMAIL: 'customersuccess@edx.org',
   TABLEAU_URL: process.env.TABLEAU_URL,
   ENTERPRISE_LEARNER_PORTAL_URL: process.env.ENTERPRISE_LEARNER_PORTAL_URL,
+  ALGOLIA: {
+    APP_ID: process.env.ALGOLIA_APP_ID,
+    SEARCH_API_KEY: process.env.ALGOLIA_SEARCH_API_KEY,
+    INDEX_NAME: process.env.ALGOLIA_INDEX_NAME,
+  },
 };
 
 const features = {
   CODE_MANAGEMENT: process.env.FEATURE_FLAGS.CODE_MANAGEMENT || hasFeatureFlagEnabled('CODE_MANAGEMENT'),
+  BULK_ENROLLMENT: process.env.FEATURE_FLAGS.BULK_ENROLLMENT || hasFeatureFlagEnabled('BULK_ENROLLMENT'),
```

Comment:
Yeah, so this is the only place you need the feature flag; in .env variables. Heads up the env variable process is going to change a bit today when I merge my PR.
There's no need to add this to the backend; we don't need enterprises to be able to set or unset this feature. When we're done with it, we'll remove the flag and everyone will have it.